hack/update-kubevirtci.sh

```diff
@@ -17,7 +17,7 @@
 SCRIPT_ROOT="$(cd "$(dirname $0)/../" && pwd -P)"
 
 # the kubevirtci tag to vendor from (https://github.com/kubevirt/kubevirtci/tags)
-kubevirtci_release_tag=2303092154-4b9b02c
+kubevirtci_release_tag=2304141928-32ab159
```

Comment:
maybe safer to go with something a little older & integrated to kubevirt/kubevirt?
https://github.com/kubevirt/kubevirt/pull/9554

As you can see in https://github.com/kubevirt/kubevirt/pull/9616 `2304141928-32ab159` doesn't look very promising

Reply:
Sure, let's try our luck with `2303311039-054c0ed`

Reply:
did you also run the `hack/update-kubevirtci.sh` script? because when I compare the new push I don't see any changes to cluster-up

Reply:
yes, the only relevant change is image tags apparently